sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/connection/OracleConnectionProvider.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.datasources.jdbc.connection
+
+import java.security.PrivilegedExceptionAction
+import java.sql.{Connection, Driver}
+import java.util.Properties
+
+import org.apache.hadoop.security.UserGroupInformation
+
+import org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions
+
+private[sql] class OracleConnectionProvider(driver: Driver, options: JDBCOptions)
+    extends SecureConnectionProvider(driver, options) {
```

Comment:
nit. Indentation?
- https://github.com/databricks/scala-style-guide/blame/master/README.md#L242

Reply:
Sorry for nit-picking. You are a senior and good contributor but it seems that you have been not aware of the Apache Spark community guide in this specific example. I saw similar instances in this series of PR. So, I added the above comment. Your commits an PRs are already referenced a lot. :) Thanks.

Reply:
Looked back from where this can come from. The only thing what I can think of is mixing up parameter 4 space indent w/ extends 2 space :) I'll consider this more from now on...

Reply:
As you've pinpointed all other providers suffer from the same issue. In order keep focus I'm intended to file a minor PR to solve them separately. This one fixed here.

Reply:
https://github.com/apache/spark/pull/28945